Jenkinsfile

```diff
@@ -1418,18 +1353,5 @@ pipeline {
                 }
             }
         }
-
-
-        // Single functional test
-        stage('PROD - Smoke Tests') {
```

Comment:
Do the Smoke Tests need to be removed? It seems like a smoke test would be helpful to keep.